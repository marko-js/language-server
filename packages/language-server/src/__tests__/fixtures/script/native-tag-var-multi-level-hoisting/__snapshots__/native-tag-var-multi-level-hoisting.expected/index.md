## Hovers
### Ln 3, Col 8
```marko
  1 | <div>
  2 |   <div onClick() {
> 3 |     searchInput().focus();
    |        ^ const searchInput: never
  4 |     // ^?
  5 |   }>
  6 |     text
```

## Diagnostics
### Ln 3, Col 5
```marko
  1 | <div>
  2 |   <div onClick() {
> 3 |     searchInput().focus();
    |     ^^^^^^^^^^^ This expression is not callable.
  Type 'never' has no call signatures.
  4 |     // ^?
  5 |   }>
  6 |     text
```

