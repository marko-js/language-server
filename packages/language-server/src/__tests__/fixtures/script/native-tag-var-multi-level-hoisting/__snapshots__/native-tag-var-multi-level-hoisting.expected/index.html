<div data-marko-node-id="0">
  <div data-marko-node-id="1" onClick>
    text
    <input data-marko-node-id="2" type="search">
  </div>
</div>